build.gradle.kts

```diff
@@ -238,4 +238,6 @@ subprojects {
 
         sign(publishing.publications["mavenArtifacts"])
     }
-}
\ No newline at end of file
+}
+
+defaultTasks("format", "static", "check")
```

Comment:
This means you can just run `./gradlew` from the terminal to format and check the code.

(Still do a `./gradlew build` before committing as that'll also check javadoc etc.)

Reply:
no worries